### PR TITLE
feat: crc token total supply

### PR DESCRIPTION
**Pull request by vibern0**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -105,7 +105,7 @@ type Avatar {
   avatarType: AvatarType!
   invitedBy: String
   # v1 token address or equal to avatar address if v2
```
Comment (teawaterwire) [inline schema.graphql]: this comment no longer valid then?

Comment (teawaterwire) [reply schema.graphql]: (not an address, but an entity i mean)

File: src/common/handleTransfer.ts [hunk 2]
```diff
@@ -81,7 +89,24 @@ export const handleTransfer = async ({
         version,
         tokenType: mapAvatarTypeToTokenType(avatarType),
         tokenOwner_id,
+        totalSupply,
       });
+    } else {
+      let totalSupply = 0n;
```
Comment (teawaterwire) [inline src/common/handleTransfer.ts]: isn't it odd that you set it to 0 here? if the Token exists?

File: schema.graphql (hunk 1 above)
Comment (vibern0) [reply schema.graphql]: right

File: src/common/handleTransfer.ts (hunk 2 above)
Comment (vibern0) [reply src/common/handleTransfer.ts]: oops correct

**Pull request merged**